make-distribution.sh

```diff
@@ -121,7 +121,7 @@ if [ $(command -v git) ]; then
 fi
 
 
-if [ ! $(command -v "$MVN") ] ; then
+if [ ! "$(command -v "$MVN")" ] ; then
```

Comment:
It does, here is the output from that line when it has quotes:
+ '[' '!' '/tmp/test spaces/spark/build/mvn' ']'


Before:
+ '[' '!' /tmp/test spaces/spark/build/mvn ']'